.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
Maybe we can include the "Run in Chrome" config for the connect widget?

Something like:

```json
        {
          "name": "Run Widget in Chrome",
          "type": "chrome",
          "request": "launch",
          "url": "http://localhost:3000",
          "webRoot": "${workspaceFolder}/connect-widget/app/"
        }
```